package.json

```diff
@@ -47,6 +53,7 @@
     "esbuild-plugin-sass": "^1.0.1",
     "esbuild-rails": "^1.0.3",
     "file-loader": "^6.2.0",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
why do we need this on dependencies ?

Reply:
[Errorr](https://gist.github.com/pulkit-30/9182f6875af1239515e1cc4a481b6f25#error---css-files)  for `importing CSS modules` -- `codemirror.css`

Error log:

```
    Details:

    /Users/pulkitgupta/Desktop/OpenSource/CircuitVerse/node_modules/codemirror/lib/codemirror.css:3
    .CodeMirror {
    ^

    SyntaxError: Unexpected token '.'

      3 |
      4 | import CodeMirror from 'codemirror/lib/codemirror.js';
    > 5 | import 'codemirror/lib/codemirror.css';
        | ^
      6 |
      7 | // Importing CodeMirror themes
      8 | import 'codemirror/theme/3024-day.css';

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1773:14)
      at Object.<anonymous> (simulator/src/Verilog2CV.js:5:1)
```
[identity-obj-proxy](https://github.com/keyz/identity-obj-proxy#identity-obj-proxy---) solves this issue (simply say mock these modules while running test).

### Jest Config

```
.
.

    moduleNameMapper: {
        "\\.(css|less|sass|scss)$": "identity-obj-proxy",
        "typeface-nunito": "identity-obj-proxy",
    },
.
.

```


Open to an alternative solution